Set up user docs for RisingWave Cloud (#596)
risingwavelabs/risingwave-docs: cloud/cluster-manage-database-users.md

---
id: cluster-manage-database-users
title: Manage database users
description: A database user allows a person or application to access a specific database or set of databases within a cluster.
slug: /manage-database-users
---

> In RisingWave Database, a database user is similar to a database user/role in Postgres, allowing a person or application to access a specific database or set of databases within a cluster. Also, you [connect and log in to a cluster](cluster-connect-to-a-cluster.md) as one of the database users in it. Database users can be assigned specific permissions to control their database privileges.

Reviewer: "user or role"

> In RisingWave Database, a database user is similar to a database user or role in Postgres, allowing a person or application to access a specific database or set of databases within a cluster. Also, you [connect and log in to a cluster](cluster-connect-to-a-cluster.md) as one of the database users in it. Database users can be assigned specific permissions to control their database privileges.

Reviewer 2: ```suggestion
In RisingWave Database, a database user is similar to a database user or role in Postgres, allowing a person or application to access a specific database or set of databases within a cluster. Also, you [connect and log in to a cluster](cluster-connect-to-a-cluster.md) as one of its database users. Database users can be assigned specific permissions to control their database privileges.
```
